integration-test/cli.Tests.ps1

```diff
@@ -155,12 +151,14 @@ Describe 'MAUI' -ForEach @(
             'libxamarin-app.so',
             'maui-app.pdb'
         )
+        $nonZeroNumberRegex = '[1-9][0-9]*';
         $result.ScriptOutput | Should -AnyElementMatch 'Uploaded a total of 1 new mapping files'
-        $result.ScriptOutput | Should -AnyElementMatch 'Found 25 debug information files \(1 with embedded sources\)'
+        $result.ScriptOutput | Should -AnyElementMatch "Found $nonZeroNumberRegex debug information files \($nonZeroNumberRegex with embedded sources\)"
```

Comment:
Wouldn't we want the hard coded value instead?
It shouldn't change once we validate the value is correct, right? If instead of 25 we get 1, that'd prob be broken.

Reply:
I've reverted to hard coding the expectations... that way at least we'll detect when they change and can decide whether to be worried about it or not.